docs/2.podidentity.md

```diff
@@ -26,7 +26,7 @@ HIGHLY RECOMMENDED: Verify pod identity is operational on each AKS cluster.
 1. Obtain Subscription ID, AKS Resource Group name, AKS name, and Client ID of each pod identity:
 ```bash
 az account show --query "{subscriptionId: id}" -o table
-az aks list --query '[].{resourceGroup:resourceGroup, AksName:name, clientId: podIdentityProfile.userAssignedIdentities[0].identity.clientId}' -o table
+

```

Comment:
Looks like the incoming change actually removed the line. Can you add it back in and then I'll approve/merge the PR.